lib/Conversion/TritonGPUToLLVM/TypeConverter.cpp

```diff
@@ -13,6 +13,19 @@ using ::mlir::triton::gpu::MmaEncodingAttr;
 using ::mlir::triton::gpu::SharedEncodingAttr;
 using ::mlir::triton::gpu::SliceEncodingAttr;
 
+namespace {
+
+// Get number of elements per thread for $a and $b operand of MFMA dot op.
+static size_t getMFMAOperandNumElemsPerThread(RankedTensorType operand) {
+  auto encoding = operand.getEncoding().cast<DotOperandEncodingAttr>();
+  auto elemTy = operand.getElementType();
+  auto reps = encoding.getMMAv3Rep(operand.getShape(), elemTy);
+  return reps[0] * reps[1];
+}
+
+} // namespace
+
+
```

Comment:
Perhaps `#ifdef USE_ROCM` is needed here too?
And one of two empty lines the the end seems to be redundant.